app/views/govuk_publishing_components/components/_contents_list.html.erb

```diff
@@ -35,7 +37,11 @@
         title_fallback == "en" ? {:lang => "en",} : {}
       )
     ) do %>
-      <%= t("components.contents_list.contents") %>
+      <% if alternate_title.empty? %>
```

Comment:
This section can be written more simply as below, if you amend the original variable declaration...

```Ruby
alternate_title ||= nil

# ...

<%= alternate_title || t("components.contents_list.contents") %>
```

Reply:
And I think with this change you can also shorten line 28 to 

```Ruby
component_helper.add_class("gem-c-contents-list--alternate-title") if alternate_title
```

Reply:
Nice, thanks for sharing that. I'm still learning these shortcuts. I've updated it as you described, along with the changed variable names from the other comments.